app/views/homepage/index.html.erb

```diff
@@ -229,3 +229,11 @@
     </div>
   </div>
 </main>
+<% # This is for JS/non-JS tracking purposes %>
```

Comment:
Should we make this a HTML comment so view-source adventurers know why we're doing shading looking tracking?